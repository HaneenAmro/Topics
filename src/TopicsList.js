import Accordion from '@material-ui/core/Accordion';
import AccordionSummary from '@material-ui/core/AccordionSummary';
import AccordionDetails from '@material-ui/core/AccordionDetails';
import Typography from '@material-ui/core/Typography';
import RelatedTopics from './RelatedTopic.js';
import styled from 'styled-components';


const Container = styled.div`
    width :70%;
    margin-left : 14%;;
    margin-bottom : 3%;
`;

const Span = styled.span`
    display: flex;
    justify-content: space-between;
    text-decoration: underline double green;
    text-underline-position: under;
`;

//This function to render the main topics the relate to react and each topic has nested topics
function TopicList(props) {
    return (
        <Container>
        <Accordion>
            <AccordionSummary>
                <Typography>
                    <Span><strong>Topic Name:</strong>{props.name} </Span>
                    <Span><strong>Stargazer count:</strong>{props.count} </Span>
                </Typography>
            </AccordionSummary>
            <AccordionDetails>
                <Typography>
                    <div>
                    <strong>Related Topics:</strong>
                        {props.related.map((topic) => (
                            < RelatedTopics relName = {topic.name} relCount = {topic.stargazerCount}  />
                        ))}
                    </div>
                </Typography>
            </AccordionDetails>
        </Accordion>
        </Container>
    )
}

export default TopicList;