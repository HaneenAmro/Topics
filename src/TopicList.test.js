
import renderer from 'react-test-renderer';
import TopicsList from './TopicsList.js';

describe('My compnent', () => {
    const testProps =
    {
        "data": {
            "topic": {
                "relatedTopics": [
                    {
                        "id": "MDU6VG9waWNhbmd1bGFy",
                        "name": "angular",
                        "stargazerCount": 31274,
                        "relatedTopics": [
                            {
                                "name": "react",
                                "stargazerCount": 51339
                            },
                            {
                                "name": "vue",
                                "stargazerCount": 37685
                            },
                            {
                                "name": "azure-functions",
                                "stargazerCount": 15
                            },
                            {
                                "name": "frontend",
                                "stargazerCount": 21594
                            },
                            {
                                "name": "backend",
                                "stargazerCount": 276
                            },
                            {
                                "name": "typescript",
                                "stargazerCount": 18711
                            },
                            {
                                "name": "asp",
                                "stargazerCount": 61
                            },
                            {
                                "name": "cli",
                                "stargazerCount": 16199
                            },
                            {
                                "name": "firebase",
                                "stargazerCount": 15240
                            },
                            {
                                "name": "spring-boot",
                                "stargazerCount": 16830
                            }
                        ]
                    },
                ]
            }
        }
    }



                    it('it works', () => {
                        const tree = renderer.create(<TopicsList key={testProps.data.topic.relatedTopics[0].id}  name={testProps.data.topic.relatedTopics[0].name} count={testProps.data.topic.relatedTopics[0].stargazerCount} 
                            related = {testProps.data.topic.relatedTopics[0].relatedTopics} />).toJSON();
                        expect(tree).toMatchSnapshot();
                    });
            });
