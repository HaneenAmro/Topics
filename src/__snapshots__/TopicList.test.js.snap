// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My compnent it works 1`] = `
<div
  className="sc-dlnjwi ipSvBe"
>
  <div
    className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      aria-disabled={false}
      aria-expanded={false}
      className="MuiButtonBase-root MuiAccordionSummary-root"
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <div
        className="MuiAccordionSummary-content"
      >
        <p
          className="MuiTypography-root MuiTypography-body1"
        >
          <span
            className="sc-hKFxyN bKHOAc"
          >
            <strong>
              Topic Name:
            </strong>
            angular
             
          </span>
          <span
            className="sc-hKFxyN bKHOAc"
          >
            <strong>
              Stargazer count:
            </strong>
            31274
             
          </span>
        </p>
      </div>
    </div>
    <div
      className="MuiCollapse-container MuiCollapse-hidden"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper"
      >
        <div
          className="MuiCollapse-wrapperInner"
        >
          <div
            role="region"
          >
            <div
              className="MuiAccordionDetails-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1"
              >
                <div>
                  <strong>
                    Related Topics:
                  </strong>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              react
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    51339
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              vue
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    37685
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              azure-functions
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    15
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              frontend
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    21594
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              backend
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    276
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              typescript
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    18711
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              asp
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    61
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              cli
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    16199
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              firebase
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    15240
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="sc-bdnxRM eXgWxp"
                  >
                    <div
                      className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
                    >
                      <div
                        aria-disabled={false}
                        aria-expanded={false}
                        className="MuiButtonBase-root MuiAccordionSummary-root"
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        role="button"
                        tabIndex={0}
                      >
                        <div
                          className="MuiAccordionSummary-content"
                        >
                          <p
                            className="MuiTypography-root MuiTypography-body1"
                          >
                            <span
                              className="sc-gtsrHT gxLBAN"
                            >
                              <strong>
                                Topic Name:
                              </strong>
                              spring-boot
                            </span>
                          </p>
                        </div>
                      </div>
                      <div
                        className="MuiCollapse-container MuiCollapse-hidden"
                        style={
                          Object {
                            "minHeight": "0px",
                          }
                        }
                      >
                        <div
                          className="MuiCollapse-wrapper"
                        >
                          <div
                            className="MuiCollapse-wrapperInner"
                          >
                            <div
                              role="region"
                            >
                              <div
                                className="MuiAccordionDetails-root"
                              >
                                <p
                                  className="MuiTypography-root MuiTypography-body1"
                                >
                                  <span
                                    className="sc-gtsrHT gxLBAN"
                                  >
                                    <strong>
                                      Stargazer count:
                                    </strong>
                                    16830
                                  </span>
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
